src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/OrganizationState.scala

```diff
@@ -0,0 +1,63 @@
+package ch.epfl.bluebrain.nexus.admin.organizations
+import java.time.Instant
+
+import ch.epfl.bluebrain.nexus.admin.config.AppConfig.HttpConfig
+import ch.epfl.bluebrain.nexus.admin.config.Vocabulary.nxv
+import ch.epfl.bluebrain.nexus.admin.types.{ResourceF, ResourceMetadata}
+import ch.epfl.bluebrain.nexus.commons.types.identity.Identity
+
+/**
+  * Enumeration of organization states.
+  */
+sealed trait OrganizationState
```

Comment:
`... extends Product with Serializable` to avoid the LUB being different from this trait